pkg/controller/migration/namespace_migration.go

```diff
@@ -456,6 +540,14 @@ func (m *CoreNamespaceMigration) migrateEachNode(ctx context.Context, log logr.L
 				log.WithValues("error", err).V(1).Info("Error checking for new healthy pods")
 				time.Sleep(10 * time.Second)
 			}
+
+			// Wait for the operator-managed Typha deployment to be ready.
+			if err := m.waitForOperatorTyphaDeploymentReady(ctx, log); err != nil {
+				return fmt.Errorf("failed to wait for operator typha deployment to be ready: %s", err.Error())
+			}
+			log.V(1).Info("calico-system/calico-typha is running with expected replica count after migrating node")
```

Comment:
If it would then I think you should increase the level for this one because I don't think for normal operation we need 2 different messages on each node.

Reply:
V(1) is not included in the default logging level